cli/src/utils/toolchain.ts

```diff
@@ -47,18 +51,18 @@ function createPathFunction({
     await downloadFunction({
       localFilePath,
       dirPath,
-      ...downloadParams
+      ...downloadParams,
```

Comment:
Nice. Was that pointed out by some linter or is it a manual change? Would be nice to catch such stuff in CI.

Reply:
the linter did it automatically! I am used to setup visualstudiocode with linters/autoformatters and autosave :) we'd probably standardize it

Reply:
Fair enough. It would be nice to eventually use the same linter in `scripts/lint.sh`, but maybe let's do that in a separate PR

Reply:
yeah, good tip, we can standardise it later